tests/psalm-baseline.xml

```diff
@@ -29,6 +29,36 @@
       <code>\OC_Util</code>
     </UndefinedClass>
   </file>
+  <file src="lib/Command/Bot/Install.php">
+    <UndefinedClass>
+      <code>Base</code>
+    </UndefinedClass>
+  </file>
```

Comment:
No stub file?

Reply:
Not there for all the others since a long time. I'll raise an issue.

Reply:
https://github.com/nextcloud/spreed/issues/10132